pkg/blockatlas/observer.go

```diff
@@ -0,0 +1,17 @@
+package blockatlas
+
+type Webhook struct {
+	Subscriptions     map[string][]string `json:"subscriptions"`
+	XpubSubscriptions map[string][]string `json:"xpub_subscriptions"`
```

Comment:
`XpubSubscriptions map[string][]string `json:"xpub_subscriptions"`` let's remove this

Reply:
yes! I going to do that in the next PR, I only need this to finish refactor the notifications inside the backend